libcnb-package/src/cross_compile.rs

```diff
@@ -7,95 +8,71 @@ use which::which;
-                            OsString::from("CC_x86_64_unknown_linux_musl"),
-                            OsString::from(&gcc_binary_path),
+                            OsString::from(format!("CC_{}", target.replace('-', "_"))),
+                            OsString::from(gcc_path),
                         ),
                     ],
                 }
-            })
-    } else if target_triple.as_ref() == X86_64_UNKNOWN_LINUX_MUSL && cfg!(target_os = "linux") {
-        match which("musl-gcc") {
-            Ok(_) => CrossCompileAssistance::Configuration {
-                cargo_env: Vec::new(),
-            },
-            Err(_) => CrossCompileAssistance::HelpText(String::from(
-                r"For cross-compilation from Linux to x86_64-unknown-linux-musl, a C compiler and
-linker for the target platform must be installed on your computer.
-
-The easiest way to install 'musl-gcc' is to install the 'musl-tools' package:
-- https://packages.ubuntu.com/focal/musl-tools
-- https://packages.debian.org/bullseye/musl-tools
-
-You will also need to install the Rust target, using:
-rustup target add x86_64-unknown-linux-musl",
-            )),
+            }
         }
-    } else if target_triple.as_ref() == AARCH64_UNKNOWN_LINUX_MUSL && cfg!(target_os = "linux") {
-        let gcc_binary_path = "aarch64-linux-gnu-gcc";
-        match which(gcc_binary_path) {
-            Ok(_) => CrossCompileAssistance::Configuration {
-                cargo_env: vec![
-                    (
-                        OsString::from("CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER"),
-                        OsString::from(gcc_binary_path),
-                    ),
-                    (
-                        OsString::from("CC_aarch64_unknown_linux_musl"),
-                        OsString::from(gcc_binary_path),
-                    ),
-                ],
-            },
-            Err(_) => CrossCompileAssistance::HelpText(String::from(
-                r"For cross-compilation from Linux to aarch64-unknown-linux-musl, a C compiler and
-linker for the target platform must installed on your computer.
-
-The easiest way to install the 'g++-aarch64-linux-gnu', 'libc6-dev-arm64-cross', and 'musl-tools' packages:
-- https://packages.ubuntu.com/focal/g++-aarch64-linux-gnu
-- https://packages.ubuntu.com/focal/musl-tools
-- https://packages.ubuntu.com/focal/libc6-dev-arm64-cross
+        Err(_) => CrossCompileAssistance::HelpText(format!(
+            r"For cross-compilation from {0} {1} to {target}, a C compiler and
+linker for the target platform must be installed:
 
-You will also need to install the Rust target, using:
-rustup target add aarch64-unknown-linux-musl",
-            )),
-        }
-    } else {
-        CrossCompileAssistance::NoAssistance
+{help_text}
+            
+You will also need to install the Rust target:
+rustup target add {target}",
+            consts::ARCH,
+            consts::OS
+        )),
```

Comment:
This seems like a good use-case for [indoc](https://docs.rs/indoc/latest/indoc/index.html). We use that crate a lot and it helps making these multi-line strings much nicer in the code as it will strip away the common whitespace for your. And since it's a macro, there is no impact on the resulting code.